trunk/ghost/ghostparse.c

```diff
@@ -91,27 +35,23 @@ static void Ghost_ListToArray(ghostreclist_t *list, ghostrec_t **records_out,
 
 
 typedef struct {
+    ghost_info_t *ghost_info;
+    ghost_level_t *level;
+
     int view_entity;
-    ghostreclist_t **next_ptr;
     ghostrec_t rec;
-    qboolean map_found;
     qboolean updated;  // record changed since last append
 
     FILE *demo_file;
 
     int model_num;
-    const char *expected_map_name;
 
     vec3_t baseline_origin;
     vec3_t baseline_angle;
     int baseline_frame;
     int baseline_model;
 
-    char (*client_names)[MAX_SCOREBOARDNAME];
-    byte *client_colors;
-    float finish_time;
     char next_demo_path[MAX_OSPATH];
-    int model_indices[GHOST_MODEL_COUNT];
```

Comment:
These are written directly into the `ghost_info_t` now.  There's no benefit to the extra copy.